### PR TITLE
Multiple bugfixes

### PR DESCRIPTION
**Pull request by kunalspathak**
Please refer to individual commits for bug fixes.

### CODE REVIEW

File: lib/jsrt/jsrtinternal.h [hunk 1]
```diff
@@ -131,7 +131,7 @@ JsErrorCode ContextAPIWrapper(Fn fn)
     Js::ScriptContext *scriptContext = currentContext->GetScriptContext();
     try
     {
-        AUTO_NESTED_HANDLED_EXCEPTION_TYPE((ExceptionType)ExceptionType_JavascriptException);
```
Comment (Yongqu) [inline lib/jsrt/jsrtinternal.h]: Should we have same thing for stack overflow?

Comment (kunalspathak) [inline lib/jsrt/jsrtinternal.h]: Thought about that, but didn't want  to add without knowing if we will get OOS here. I will add if I see a reason to add it.

**Comment by kunalspathak** [comment]
I had to rebase after `Lib` changes. Incorporated review comment by @Yongqu

**Comment by Yongqu** [comment]
LGTM

File: Lib/Runtime/Base/WindowsGlobalizationAdapter.cpp [hunk 2]
```diff
@@ -425,19 +425,68 @@ namespace Js
         return hr;
     }
 
-    void WindowsGlobalizationAdapter::ClearTimeZoneCalendars()
```
Comment (Yongqu) [inline Lib/Runtime/Base/WindowsGlobalizationAdapter.cpp]: test. pls ignore

**Pull request merged**